gui/src/components/maintenance/cortx-about.vue

```diff
@@ -113,11 +207,26 @@ export default class Cortxaboutpage extends Vue {
         RELEASE: null,
         COMPONENTS: []
       },
+      issuerDetails:{
+        commonName:"-" as string,
+        countryName:"-" as string,
+        localityName:"-" as string,
+        organizationName:"-" as string,
+
+      },
+      subject:{
+        commonName:"-" as string,
+        countryName:"-" as string,
+        localityName:"-" as string,
+        organizationName:"-" as string,
+
+      },
       serialNumber: "-" as string
     };
   }
   
   public async mounted() {
+    await this.getSSLDetails();
```

Comment:
Promise.all should be used in case of independent async operations.

Reply:
done